From 08f5af65812dfae784080fd8ce91e934581fd1e9 Mon Sep 17 00:00:00 2001
From: Kevin Lau Fang <fangk@google.com>
Date: Thu, 14 Nov 2019 20:06:36 +0000
Subject: [PATCH 1/7] Revert submission

Reason for revert: This causes a boot issue on Taimen (b/144480969). It wasn't caught by TreeHugger due to Taimen tests being disabled due to a different lab outage (b/144350336).

Change-Id: I6b4cd2d286c44231cb8f4789b7e36633978cd060
(cherry picked from commit 1bd49c2ccdd59e76b2546cec8a4b02b66e7e7123)
---
 Settings/src/com/android/tv/settings/about/AboutFragment.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Settings/src/com/android/tv/settings/about/AboutFragment.java b/Settings/src/com/android/tv/settings/about/AboutFragment.java
index e3f7d988..43ac4c24 100644
--- a/Settings/src/com/android/tv/settings/about/AboutFragment.java
+++ b/Settings/src/com/android/tv/settings/about/AboutFragment.java
@@ -116,7 +116,7 @@ public class AboutFragment extends SettingsPreferenceFragment {
         PreferenceUtils.resolveSystemActivityOrRemove(getActivity(), screen, deviceNamePref, 0);
 
         final Preference firmwareVersionPref = findPreference(KEY_FIRMWARE_VERSION);
-        firmwareVersionPref.setSummary(Build.VERSION.RELEASE_OR_CODENAME);
+        firmwareVersionPref.setSummary(Build.VERSION.RELEASE);
         firmwareVersionPref.setEnabled(true);
 
         final Preference securityPatchPref = findPreference(KEY_SECURITY_PATCH);
-- 
2.35.1

